pkg/controller/node/nodecontroller.go

```diff
@@ -367,14 +367,15 @@ func (nc *NodeController) Run(period time.Duration) {
 		defer nc.evictorLock.Unlock()
 		for k := range nc.zonePodEvictor {
 			nc.zonePodEvictor[k].Try(func(value TimedValue) (bool, time.Duration) {
-				remaining, err := deletePods(nc.kubeClient, nc.recorder, value.Value, nc.daemonSetStore)
+				uid, _ := value.UID.(string)
```

Comment:
Reading at this point, it's not clear what was put into `value`.
Consider naming this var `nodeUid` to clarify.
(same below line 392)